scss/main.scss

```diff
@@ -337,15 +374,15 @@ h1,h2,h3 {
 }
 
 .whitebutton {
-  background-color: $white;
-  border: 1px solid $cfa-red;
+  background-color: $cfa-red;
+  border-radius: 2px;
   text-align: center;
   padding-top: 0.5rem;
   padding-bottom: 0.5rem;
   padding-left: .75rem;
   padding-right: .75rem;
   a {
-    color: $cfa-red !important;
+    color: $white !important;
```

Comment:
Can remove this nested `a` selector? The `whitebutton` class is placed on an anchor tag so this style doesn't do anything.